ansible/roles/certificates/tasks/main.yml

```diff
@@ -6,6 +6,10 @@
   command: openssl req -x509 -nodes -subj "/C=FI/ST=Uusimaa/L=Espoo/O=CSC/CN={{ server_domain_name }}" -days 365 -newkey rsa:2048 -keyout {{ ssl_certificates_path }}/{{ ssl_key_name }} -out {{ ssl_certificates_path }}/{{ ssl_certificate_name }} creates={{ ssl_certificates_path }}/{{ ssl_certificate_name }}
   when: deployment_environment_id in ['local_development', 'staging']
 
+- name: Create self-signed redirect SSL cert and private key
+  command: openssl req -x509 -nodes -subj "/C=FI/ST=Uusimaa/L=Espoo/O=CSC/CN={{ redirect_server_domain_name }}" -days 365 -newkey rsa:2048 -keyout {{ ssl_certificates_path }}/{{ redirect_ssl_certificate_key_name }} -out {{ ssl_certificates_path }}/{{ redirect_ssl_certificate_name }} creates={{ ssl_certificates_path }}/{{ redirect_ssl_certificate_name }}
+  when: deployment_environment_id in ['local_development', 'test', 'stable', 'demo']
```

Comment:
I think the rule could be ``when: deployment_environment_id not in ['production']``

Reply:
Thanks. This is a better alternative.